src/Neo.VM/Types/StackItem.Vertex.cs

```diff
@@ -120,7 +125,14 @@ internal class ObjectReferenceEntry
         /// Use this method when you need a hash code for a StackItem.
         /// </summary>
         /// <returns>The hash code for the StackItem.</returns>
-        public override int GetHashCode() =>
-            HashCode.Combine(GetSpan().ToArray());
+        public override int GetHashCode()
+        {
+            if (_hashCode == 0)
+            {
+                return _hashCode = HashCode.Combine(Type, Unsafe.HashBytes(GetSpan()));
```

Comment:
If  `GetSpan`  of `StackItem` subtype is not  implemented, `GetHashCode` will throw an `Exception`.

`StackItem` subtypes should implement  its own `GetHashCode` and `Equals`.

Reply:
They already `override` the method